src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -94,6 +94,15 @@ namespace AirflowNetwork {
         UserAspectRatio  // Effective rectangle user input aspect ratio selection
     };
 
+    enum class iDuctLineType
```

Comment:
I've been making some "hygiene" comments on PRs for the last few months, so I will make them here too. A few comments about this `enum class`:

- I am not super-anti Hungarian notation but I am not pro-Hungarian either and we don't really use it elsewhere in EnergyPlus (we do use "scoping Hungarian" in some modules by using `m_` to denote member variables, but we don't really use traditional "typing Hungarian" anywhere). TL;DR remove the `i` from `iDuctLineType`.
- Add `Invalid = -1` as the first item and `Num` as the last time. I am assuming that `None` is a valid choice for this enumeration as opposed to an error. If it is an error then remove it because we are using `Invalid = -1` for that.

Reply:
I think this one is maybe because of a naming thing, but if @lgu1234 changes a variable name to `ductLineType` all will be well. I'm pretty sure that `None` is a valid choice.

Reply:
I will use ductLineType and going to make changes.

Reply:
@Myoldmopar Here are my changes:

    enum class DuctLineType
    {
        Invalid = -1,
        None,
        SupplyTrunk,  // Supply trunk
        SupplyBranch, // SupplyBrnach
        ReturnTrunk,  // Return trunk
        ReturnBranch, // ReturnBrnach
    };

        // Default Constructor
        AirflowNetworkLinkageProp()
            : AirflowNetworkLinkage(), ZoneNum(0), DetOpenNum(0), ConnectionFlag(iEPlusComponentType::Invalid), VAVTermDamper(false),
              LinkageViewFactorObjectNum(0), AirLoopNum(0), ductLineType(DuctLineType::Invalid)